DEVELOPMENT.md

```diff
@@ -256,6 +279,28 @@ To run the unit tests, simply use hatch:
 hatch run test
 ```
 
+##### Integration Tests
+Adaptor integration tests are located under `test/integ/test_maya_adaptors.py`. If you are adding
+or modifying functionality, then you will want to be writing one or more integ tests to demonstrate that your
+logic behaves as expected and that future changes do not accidentally break your change.
+
+To run the integ tests, you need to:
+1. Add `mayapy` executable to the PATH:
+   
+   For example, default path of mayapy on OSX is 
+
+```
+/Applications/Autodesk/maya<version>/Maya.app/Contents/MacOS
+```
+2. Use hatch to run adaptor tests:
+```bash
+hatch run integ:test_adaptors
+```
+3. (Optional) Use hatch to run all integ tests:
+```bash
+hatch run integ:test
+```
```

Comment:
If you indent the code blocks by 4 spaces, it should render nicely "under" each number in the list